src/odemis/acq/stream/_live.py

```diff
@@ -528,6 +528,24 @@ def _DoPrepare(self):
 #         super(AlignedSEMStream, self)._onActive(active)
 
 
+class FastEMSEMStream(SEMStream):
+    """
+    SEM stream with special pixelsize VA (driver value is adjusted with scale).
+    """
+
+    def __init__(self, name, detector, dataflow, emitter, blanker=None, **kwargs):
+        super().__init__(name, detector, dataflow, emitter, blanker=None, **kwargs)
+        self.pixelSize = model.VigilantAttribute(0, unit="m", readonly=True)
```

Comment:
pixelSize is normally a tuple. It'll be confusing to have in one place one VA which doesn't follow the same convention. So I'd suggest to either make it a tuple, and change the way the GUI displays it, or rename it (maybe "pixelSize1D", pixelSizeOne, pixelSizeSingle, or pixelSizeUI). You can then change the "label" in the conf.data.